src/blip/Message.cc

```diff
@@ -343,7 +343,7 @@ namespace litecore { namespace blip {
 
     Error MessageIn::getError() const {
         if (!isError())
-            return {};
+            return {nullslice, 0, nullslice};
```

Comment:
Is there a MSVC compiler option to allow omitted items? It's nice not having to fill in all the blanks with zeroes...